rapids-cmake/cpm/versions.json

```diff
@@ -13,7 +13,7 @@
       "git_tag" : "12d13bdc5e74801645eba3e46a64081b9b020dca"
     },
     "nvcomp" : {
-      "version" : "2.3",
+      "version" : "2.3.1",
       "git_url" : "https://github.com/NVIDIA/nvcomp.git",
       "git_tag" : "v2.2.0",
```

Comment:
1. Why does this not match the `version`?
2. It doesn't look like a tag for `v2.3.1` exists yet. https://github.com/NVIDIA/nvcomp/tags

Reply:
If this is related to the changes in source availability, we may want to avoid misleading users by posting the git repo URL and tag. If so, can those fields be removed?

Reply:
The `version` field is used to find the maximum latest version we support. So the 2.3.X series via binary download. If no binary download is usable ( arm platforms ) we fallback to building from source which means the 2.2 series as it is the last version with public sources.

Reply:
Great, thanks for the explanation. It would be nice to have a comment explaining that, but JSON doesn't support inline comments (at least in the official spec).